diff --git a/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php b/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
index b1d46cb..0f027a6 100644
--- a/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
+++ b/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
@@ -23,7 +23,6 @@ class LoadUsers implements FixtureInterface, ContainerAwareInterface
         $admin->setUsername('admin');
         $admin->setPassword($this->encodePassword($admin, 'adminpass'));
         $admin->setRoles(array('ROLE_ADMIN'));
-        $admin->setIsActive(false);
         $manager->persist($admin);
 
         // the queries aren't done until now
