diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index c1bcff1..5685844 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -18,7 +18,7 @@
                 'bundles/event/css/events.css'
                 'bundles/event/css/main.css'
                 filter='cssrewrite'
-                filter='uglifycss'
+                filter='?uglifycss'
                 output='css/built/layout.css'
             %}
                 <link rel="stylesheet" href="{{ asset_url }}" />
