diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index bafd28d..0071555 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -3,7 +3,11 @@
     <head>
         <meta charset="UTF-8" />
         <title>{% block title %}Welcome!{% endblock %}</title>
-        {% block stylesheets %}{% endblock %}
+        {% block stylesheets %}
+            <link rel="stylesheet" href="{{ asset('bundles/event/css/event.css') }}" />
+            <link rel="stylesheet" href="{{ asset('bundles/event/css/events.css') }}" />
+            <link rel="stylesheet" href="{{ asset('bundles/event/css/main.css') }}" />
+        {% endblock %}
         <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
     </head>
     <body>
diff --git a/src/Yoda/EventBundle/Resources/views/layout.html.twig b/src/Yoda/EventBundle/Resources/views/layout.html.twig
index aaabbe3..54553fc 100644
--- a/src/Yoda/EventBundle/Resources/views/layout.html.twig
+++ b/src/Yoda/EventBundle/Resources/views/layout.html.twig
@@ -2,6 +2,4 @@
 
 {% block title 'Events' %}
 
-{% block stylesheets %}
-    <link rel="stylesheet" href="???" />
-{% endblock %}
\ No newline at end of file
+
