diff --git a/app/config/config.yml b/app/config/config.yml
index 29620f1..464557e 100644
--- a/app/config/config.yml
+++ b/app/config/config.yml
@@ -1,6 +1,7 @@
 imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
+    - { resource: "@EventBundle/Resources/config/services.yml" }
 
 framework:
     #esi:             ~
diff --git a/src/Yoda/EventBundle/DependencyInjection/Configuration.php b/src/Yoda/EventBundle/DependencyInjection/Configuration.php
deleted file mode 100644
index a3c4889..0000000
--- a/src/Yoda/EventBundle/DependencyInjection/Configuration.php
+++ /dev/null
@@ -1,29 +0,0 @@
-<?php
-
-namespace Yoda\EventBundle\DependencyInjection;
-
-use Symfony\Component\Config\Definition\Builder\TreeBuilder;
-use Symfony\Component\Config\Definition\ConfigurationInterface;
-
-/**
- * This is the class that validates and merges configuration from your app/config files
- *
- * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
- */
-class Configuration implements ConfigurationInterface
-{
-    /**
-     * {@inheritDoc}
-     */
-    public function getConfigTreeBuilder()
-    {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('event');
-
-        // Here you should define the parameters that are allowed to
-        // configure your bundle. See the documentation linked above for
-        // more information on that topic.
-
-        return $treeBuilder;
-    }
-}
diff --git a/src/Yoda/EventBundle/DependencyInjection/EventExtension.php b/src/Yoda/EventBundle/DependencyInjection/EventExtension.php
deleted file mode 100644
index 3fb41a3..0000000
--- a/src/Yoda/EventBundle/DependencyInjection/EventExtension.php
+++ /dev/null
@@ -1,28 +0,0 @@
-<?php
-
-namespace Yoda\EventBundle\DependencyInjection;
-
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\Config\FileLocator;
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
-use Symfony\Component\DependencyInjection\Loader;
-
-/**
- * This is the class that loads and manages your bundle configuration
- *
- * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
- */
-class EventExtension extends Extension
-{
-    /**
-     * {@inheritDoc}
-     */
-    public function load(array $configs, ContainerBuilder $container)
-    {
-        $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
-
-        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('services.yml');
-    }
-}
