diff --git a/src/Yoda/UserBundle/Entity/User.php b/src/Yoda/UserBundle/Entity/User.php
index 16f55ee..379c31b 100644
--- a/src/Yoda/UserBundle/Entity/User.php
+++ b/src/Yoda/UserBundle/Entity/User.php
@@ -280,6 +280,8 @@ class User implements AdvancedUserInterface, Serializable
     {
         $this->plainPassword = $plainPassword;
 
+        $this->setPassword(null);
+
         return $this;
     }
 
