diff --git a/src/Yoda/UserBundle/Doctrine/UserListener.php b/src/Yoda/UserBundle/Doctrine/UserListener.php
index 93489dc..d6286e6 100644
--- a/src/Yoda/UserBundle/Doctrine/UserListener.php
+++ b/src/Yoda/UserBundle/Doctrine/UserListener.php
@@ -2,10 +2,24 @@
 
 namespace Yoda\UserBundle\Doctrine;
 
+use Yoda\UserBundle\Entity\User;
+
 class UserListener
 {
     public function prePersist()
     {
         die('Something is being inserted!');
     }
+
+    private function handleEvent(User $user)
+    {
+        $plainPassword = $user->getPlainPassword();
+
+        $encoder = $this->container->get('security.encoder_factory')
+            ->getEncoder($user)
+        ;
+
+        $password = $encoder->encodePassword($plainPassword, $user->getSalt());
+        $user->setPassword($password);
+    }
 }
