diff --git a/src/Yoda/UserBundle/Form/RegisterFormType.php b/src/Yoda/UserBundle/Form/RegisterFormType.php
index 89036c9..169c38b 100644
--- a/src/Yoda/UserBundle/Form/RegisterFormType.php
+++ b/src/Yoda/UserBundle/Form/RegisterFormType.php
@@ -12,7 +12,9 @@ class RegisterFormType extends AbstractType
     {
         $builder
             ->add('username', 'text', array(
-                'required' => false
+                'required' => false,
+                'label' => 'Nickname',
+                'attr'    => array('class' => 'foob')
             ))
             ->add('email', 'email')
             ->add('plainPassword', 'repeated', array(
