diff --git a/src/Yoda/UserBundle/Entity/User.php b/src/Yoda/UserBundle/Entity/User.php
index d44c79b..baac841 100644
--- a/src/Yoda/UserBundle/Entity/User.php
+++ b/src/Yoda/UserBundle/Entity/User.php
@@ -45,7 +45,9 @@ class User implements AdvancedUserInterface, Serializable
     private $password;
 
     /**
-     * @ORM\Column(type="string", length=255, nullable=false)
+     * @ORM\Column(type="string", length=255)
+     * @Assert\NotBlank
+     * @Assert\Email
      */
     private $email;
 
diff --git a/src/Yoda/UserBundle/Form/RegisterFormType.php b/src/Yoda/UserBundle/Form/RegisterFormType.php
index 4d9bea0..169c38b 100644
--- a/src/Yoda/UserBundle/Form/RegisterFormType.php
+++ b/src/Yoda/UserBundle/Form/RegisterFormType.php
@@ -16,7 +16,7 @@ class RegisterFormType extends AbstractType
                 'label' => 'Nickname',
                 'attr'    => array('class' => 'foob')
             ))
-            ->add('email')
+            ->add('email', 'email')
             ->add('plainPassword', 'repeated', array(
                 'type' => 'password',
             ))
