diff --git a/src/Yoda/UserBundle/Form/RegisterFormType.php b/src/Yoda/UserBundle/Form/RegisterFormType.php
index aaad13d..6d36502 100644
--- a/src/Yoda/UserBundle/Form/RegisterFormType.php
+++ b/src/Yoda/UserBundle/Form/RegisterFormType.php
@@ -12,7 +12,9 @@ class RegisterFormType extends AbstractType
     {
         $builder
             ->add('username', 'text')
-            ->add('email', 'email')
+            ->add('email', 'email', array(
+                'required' => false
+            ))
             ->add('plainPassword', 'repeated', array(
                 'type' => 'password',
             ))
