diff --git a/src/Yoda/EventBundle/DataFixtures/ORM/LoadEvents.php b/src/Yoda/EventBundle/DataFixtures/ORM/LoadEvents.php
new file mode 100644
index 0000000..fb7d33f
--- /dev/null
+++ b/src/Yoda/EventBundle/DataFixtures/ORM/LoadEvents.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace Yoda\EventBundle\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class LoadEvents implements FixtureInterface
+{
+    public function load(ObjectManager $manager)
+    {
+        // ... todo
+    }
+}
