diff --git a/src/Yoda/UserBundle/Form/RegisterFormType.php b/src/Yoda/UserBundle/Form/RegisterFormType.php
index 2ea5b9d..aaad13d 100644
--- a/src/Yoda/UserBundle/Form/RegisterFormType.php
+++ b/src/Yoda/UserBundle/Form/RegisterFormType.php
@@ -3,9 +3,29 @@
 namespace Yoda\UserBundle\Form;
 
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
 class RegisterFormType extends AbstractType
 {
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('username', 'text')
+            ->add('email', 'email')
+            ->add('plainPassword', 'repeated', array(
+                'type' => 'password',
+            ))
+        ;
+    }
+
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $resolver->setDefaults(array(
+            'data_class' => 'Yoda\UserBundle\Entity\User',
+        ));
+    }
+
     public function getName()
     {
         return 'user_register';
