diff --git a/src/Yoda/UserBundle/Form/RegisterFormType.php b/src/Yoda/UserBundle/Form/RegisterFormType.php
index 7c2a48d..9fda224 100644
--- a/src/Yoda/UserBundle/Form/RegisterFormType.php
+++ b/src/Yoda/UserBundle/Form/RegisterFormType.php
@@ -33,7 +33,7 @@ class RegisterFormType extends AbstractType
 
     public function finishView(FormView $view, FormInterface $form, array $options)
     {
-
+        $view['email']->vars['help'] = 'It will have an @ symbol in it';
     }
 
     public function getName()
