diff --git a/src/Yoda/UserBundle/Controller/RegisterController.php b/src/Yoda/UserBundle/Controller/RegisterController.php
index f17190e..06961ad 100644
--- a/src/Yoda/UserBundle/Controller/RegisterController.php
+++ b/src/Yoda/UserBundle/Controller/RegisterController.php
@@ -7,6 +7,7 @@ use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Component\HttpFoundation\Request;
 use Yoda\UserBundle\Entity\User;
+use Yoda\UserBundle\Form\RegisterFormType;
 
 class RegisterController extends Controller
 {
@@ -19,16 +20,7 @@ class RegisterController extends Controller
         $user = new User();
         $user->setUsername('Leia');
 
-        $form = $this->createFormBuilder($user, array(
-            'data_class' => 'Yoda\UserBundle\Entity\User',
-        ))
-            ->add('username', 'text')
-            ->add('email', 'email')
-            ->add('plainPassword', 'repeated', array(
-                'type' => 'password',
-            ))
-            ->getForm()
-        ;
+        $form = $this->createForm(new RegisterFormType(), $user);
 
         $form->handleRequest($request);
         if ($form->isValid()) {
