diff --git a/src/Yoda/UserBundle/Controller/RegisterController.php b/src/Yoda/UserBundle/Controller/RegisterController.php
index f66c950..be97b42 100644
--- a/src/Yoda/UserBundle/Controller/RegisterController.php
+++ b/src/Yoda/UserBundle/Controller/RegisterController.php
@@ -5,6 +5,7 @@ namespace Yoda\UserBundle\Controller;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
 
 class RegisterController extends Controller
 {
@@ -12,7 +13,7 @@ class RegisterController extends Controller
      * @Route("/register", name="user_register")
      * @Template
      */
-    public function registerAction()
+    public function registerAction(Request $request)
     {
         $form = $this->createFormBuilder()
             ->add('username', 'text')
@@ -23,6 +24,12 @@ class RegisterController extends Controller
             ->getForm()
         ;
 
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+
+            var_dump($form->getData());die;
+        }
+
         return array('form' => $form->createView());
     }
 }
