diff --git a/src/Yoda/UserBundle/Controller/SecurityController.php b/src/Yoda/UserBundle/Controller/SecurityController.php
index 1863ca4..98279f3 100644
--- a/src/Yoda/UserBundle/Controller/SecurityController.php
+++ b/src/Yoda/UserBundle/Controller/SecurityController.php
@@ -4,13 +4,35 @@ namespace Yoda\UserBundle\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\SecurityContextInterface;
 
 class SecurityController extends Controller
 {
     /**
      * @Route("/login", name="login_form")
      */
-    public function loginAction()
+    public function loginAction(Request $request)
     {
+        $session = $request->getSession();
+
+        // get the login error if there is one
+        if ($request->attributes->has(SecurityContextInterface::AUTHENTICATION_ERROR)) {
+            $error = $request->attributes->get(
+                SecurityContextInterface::AUTHENTICATION_ERROR
+            );
+        } else {
+            $error = $session->get(SecurityContextInterface::AUTHENTICATION_ERROR);
+            $session->remove(SecurityContextInterface::AUTHENTICATION_ERROR);
+        }
+
+        return $this->render(
+            'AcmeSecurityBundle:Security:login.html.twig',
+            array(
+                // last username entered by the user
+                'last_username' => $session->get(SecurityContextInterface::LAST_USERNAME),
+                'error'         => $error,
+            )
+        );
     }
 }
\ No newline at end of file
