diff --git a/app/AppKernel.php b/app/AppKernel.php
index 6869184..59ac710 100644
--- a/app/AppKernel.php
+++ b/app/AppKernel.php
@@ -18,6 +18,7 @@ class AppKernel extends Kernel
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new Yoda\EventBundle\EventBundle(),
             new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
+            new Yoda\UserBundle\UserBundle(),
         );
 
         if (in_array($this->getEnvironment(), array('dev', 'test'))) {
diff --git a/src/Yoda/UserBundle/UserBundle.php b/src/Yoda/UserBundle/UserBundle.php
new file mode 100644
index 0000000..2302288
--- /dev/null
+++ b/src/Yoda/UserBundle/UserBundle.php
@@ -0,0 +1,9 @@
+<?php
+
+namespace Yoda\UserBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class UserBundle extends Bundle
+{
+}
