diff --git a/src/Yoda/UserBundle/Controller/RegisterController.php b/src/Yoda/UserBundle/Controller/RegisterController.php
index 2f2c4cb..f17190e 100644
--- a/src/Yoda/UserBundle/Controller/RegisterController.php
+++ b/src/Yoda/UserBundle/Controller/RegisterController.php
@@ -24,7 +24,7 @@ class RegisterController extends Controller
         ))
             ->add('username', 'text')
             ->add('email', 'email')
-            ->add('password', 'repeated', array(
+            ->add('plainPassword', 'repeated', array(
                 'type' => 'password',
             ))
             ->getForm()
@@ -35,7 +35,7 @@ class RegisterController extends Controller
             $user = $form->getData();
 
             $user->setPassword(
-                $this->encodePassword($user, $user->getPassword())
+                $this->encodePassword($user, $user->getPlainPassword())
             );
 
             $em = $this->getDoctrine()->getManager();
diff --git a/src/Yoda/UserBundle/Resources/views/Register/register.html.twig b/src/Yoda/UserBundle/Resources/views/Register/register.html.twig
index f169b35..4409ce8 100644
--- a/src/Yoda/UserBundle/Resources/views/Register/register.html.twig
+++ b/src/Yoda/UserBundle/Resources/views/Register/register.html.twig
@@ -17,11 +17,11 @@
             {{ form_row(form.username) }}
             {{ form_row(form.email) }}
 
-            {{ form_row(form.password.first, {
+            {{ form_row(form.plainPassword.first, {
                 label: 'Password'
             }) }}
 
-            {{ form_row(form.password.second, {
+            {{ form_row(form.plainPassword.second, {
                 label: 'Repeat Password'
             }) }}
 
