diff --git a/src/Yoda/UserBundle/Entity/User.php b/src/Yoda/UserBundle/Entity/User.php
index ff3a6e5..8c635b1 100644
--- a/src/Yoda/UserBundle/Entity/User.php
+++ b/src/Yoda/UserBundle/Entity/User.php
@@ -55,6 +55,8 @@ class User implements AdvancedUserInterface, Serializable
      */
     private $isActive = true;
 
+    private $plainPassword;
+
     /**
      * Get id
      *
@@ -242,4 +244,16 @@ class User implements AdvancedUserInterface, Serializable
     {
         return $this->email;
     }
+
+    public function getPlainPassword()
+    {
+        return $this->plainPassword;
+    }
+
+    public function setPlainPassword($plainPassword)
+    {
+        $this->plainPassword = $plainPassword;
+
+        return $this;
+    }
 }
