diff --git a/src/Yoda/UserBundle/Controller/RegisterController.php b/src/Yoda/UserBundle/Controller/RegisterController.php
new file mode 100644
index 0000000..0b8aa21
--- /dev/null
+++ b/src/Yoda/UserBundle/Controller/RegisterController.php
@@ -0,0 +1,13 @@
+<?php
+
+namespace Yoda\UserBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class RegisterController extends Controller
+{
+    public function registerAction()
+    {
+        // todo
+    }
+}
