diff --git a/src/Yoda/UserBundle/Controller/RegisterController.php b/src/Yoda/UserBundle/Controller/RegisterController.php
index c91e3d0..db6d490 100644
--- a/src/Yoda/UserBundle/Controller/RegisterController.php
+++ b/src/Yoda/UserBundle/Controller/RegisterController.php
@@ -4,6 +4,7 @@ namespace Yoda\UserBundle\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 
 class RegisterController extends Controller
 {
