diff --git a/src/Yoda/UserBundle/Resources/views/Register/register.html.twig b/src/Yoda/UserBundle/Resources/views/Register/register.html.twig
index dc8140f..ffd67c2 100644
--- a/src/Yoda/UserBundle/Resources/views/Register/register.html.twig
+++ b/src/Yoda/UserBundle/Resources/views/Register/register.html.twig
@@ -16,7 +16,8 @@
 
             {{ form_row(form.username) }}
             {{ form_row(form.email) }}
-            {{ form_row(form.password) }}
+            {{ form_row(form.password.first) }}
+            {{ form_row(form.password.second) }}
 
             {{ form_rest(form) }}
 
