diff --git a/app/config/security.yml b/app/config/security.yml
index 0e2d477..fca1fc3 100644
--- a/app/config/security.yml
+++ b/app/config/security.yml
@@ -3,7 +3,7 @@ security:
         Yoda\UserBundle\Entity\User: bcrypt
 
     role_hierarchy:
-        ROLE_ADMIN:       ROLE_USER
+        ROLE_ADMIN:       [ROLE_USER, ROLE_EVENT_CREATE]
         ROLE_SUPER_ADMIN: [ROLE_USER, ROLE_ADMIN, ROLE_ALLOWED_TO_SWITCH]
 
     providers:
diff --git a/src/Yoda/EventBundle/Controller/EventController.php b/src/Yoda/EventBundle/Controller/EventController.php
index 598f000..e3c3285 100644
--- a/src/Yoda/EventBundle/Controller/EventController.php
+++ b/src/Yoda/EventBundle/Controller/EventController.php
@@ -40,7 +40,7 @@ class EventController extends Controller
     public function createAction(Request $request)
     {
         $securityContext = $this->container->get('security.context');
-        if (!$securityContext->isGranted('ROLE_USER')) {
+        if (!$securityContext->isGranted('ROLE_EVENT_CREATE')) {
             // in Symfony 2.5
             // throw $this->createAccessDeniedException('message!');
             throw new AccessDeniedException('Need ROLE_USER');
@@ -90,7 +90,7 @@ class EventController extends Controller
     public function newAction()
     {
         $securityContext = $this->container->get('security.context');
-        if (!$securityContext->isGranted('ROLE_USER')) {
+        if (!$securityContext->isGranted('ROLE_EVENT_CREATE')) {
             // in Symfony 2.5
             // throw $this->createAccessDeniedException('message!');
             throw new AccessDeniedException('Need ROLE_USER');
