diff --git a/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php b/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
index 0120982..0f027a6 100644
--- a/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
+++ b/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
@@ -19,6 +19,12 @@ class LoadUsers implements FixtureInterface, ContainerAwareInterface
         $user->setPassword($this->encodePassword($user, 'userpass'));
         $manager->persist($user);
 
+        $admin = new User();
+        $admin->setUsername('admin');
+        $admin->setPassword($this->encodePassword($admin, 'adminpass'));
+        $admin->setRoles(array('ROLE_ADMIN'));
+        $manager->persist($admin);
+
         // the queries aren't done until now
         $manager->flush();
     }
