diff --git a/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php b/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
index 9391b6c..0120982 100644
--- a/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
+++ b/src/Yoda/UserBundle/DataFixtures/ORM/LoadUsers.php
@@ -16,14 +16,22 @@ class LoadUsers implements FixtureInterface, ContainerAwareInterface
     {
         $user = new User();
         $user->setUsername('user');
-        // todo - fill in this encoded password... ya know... somehow...
-        $user->setPassword('');
+        $user->setPassword($this->encodePassword($user, 'userpass'));
         $manager->persist($user);
 
         // the queries aren't done until now
         $manager->flush();
     }
 
+    private function encodePassword(User $user, $plainPassword)
+    {
+        $encoder = $this->container->get('security.encoder_factory')
+            ->getEncoder($user)
+        ;
+
+        return $encoder->encodePassword($plainPassword, $user->getSalt());
+    }
+
     public function setContainer(ContainerInterface $container = null)
     {
         $this->container = $container;
