diff --git a/src/Yoda/UserBundle/Tests/Controller/RegisterControllerTest.php b/src/Yoda/UserBundle/Tests/Controller/RegisterControllerTest.php
index 247bdff..616efc9 100644
--- a/src/Yoda/UserBundle/Tests/Controller/RegisterControllerTest.php
+++ b/src/Yoda/UserBundle/Tests/Controller/RegisterControllerTest.php
@@ -21,5 +21,8 @@ class RegisterControllerTest extends WebTestCase
             ->attr('value')
         ;
         $this->assertEquals('Foo', $usernameVal);
+
+        // the name of our button is "Register!"
+        $form = $crawler->selectButton('Register!')->form();
     }
 }
