diff --git a/src/Yoda/EventBundle/Controller/EventController.php b/src/Yoda/EventBundle/Controller/EventController.php
index 38fa8d6..b0b5e86 100644
--- a/src/Yoda/EventBundle/Controller/EventController.php
+++ b/src/Yoda/EventBundle/Controller/EventController.php
@@ -25,6 +25,13 @@ class EventController extends Controller
      */
     public function indexAction()
     {
+        $user = $this->container
+            ->get('security.context')
+            ->getToken()
+            ->getUser()
+        ;
+        var_dump($user->getUsername());die;
+
         $em = $this->getDoctrine()->getManager();
 
         $entities = $em->getRepository('EventBundle:Event')->findAll();
