diff --git a/src/Yoda/UserBundle/Entity/UserRepository.php b/src/Yoda/UserBundle/Entity/UserRepository.php
index 268ae26..5ce118a 100644
--- a/src/Yoda/UserBundle/Entity/UserRepository.php
+++ b/src/Yoda/UserBundle/Entity/UserRepository.php
@@ -29,7 +29,13 @@ class UserRepository extends EntityRepository implements UserProviderInterface
 
     public function loadUserByUsername($username)
     {
-        // todo
+        $user = $this->findOneByUsernameOrEmail($username);
+
+        if (!$user) {
+            throw new UsernameNotFoundException('No user found for username '.$username);
+        }
+
+        return $user;
     }
 
     public function refreshUser(UserInterface $user)
